app/src/main/res/values/styles_tools.xml

```diff
@@ -20,6 +20,11 @@
         <item name="android:progressDrawable">@drawable/progress_download</item>
     </style>
 
+    <style name="ShapeAppearance.GodTools.Tools.ShapeableImageView.Shape.Circle" parent="">
```

Comment:
maybe something like: `ShapeAppearance.GodTools.Tools.Action.Favorite` where it describes what you are using the ShapeAppearance for and not just that it's a circle